src/main/java/org/jenkinsci/plugins/docker/commons/impl/RegistryKeyMaterialFactory.java

```diff
@@ -111,6 +113,14 @@ public KeyMaterial materialize() throws IOException, InterruptedException {
         }
         return new RegistryKeyMaterial(dockerConfig, new EnvVars("DOCKER_CONFIG", dockerConfig.getRemote()));
     }
+    
+    protected String registry() {
+    	if (dockerExecutable.endsWith("podman") || Boolean.parseBoolean(env.get(DOCKER_REGISTRY_HOST_ONLY, "false"))) {
```

Comment:
is this podman check ideal? Better than not having it maybe, but it will not detect if the podman-docker wrapper is used.

Reply:
I don't know anything about podman, so I am not any help here :)
But I guess if anyone comes up with something better it can be enhanced in the future.